martypy/Marty.py

```diff
@@ -167,18 +167,17 @@ def dance(self, side: str = 'right', move_time: int = 4500, blocking: Optional[b
             self.client.wait_if_required(move_time, blocking)
         return result
 
-    def celebrate(self, move_time: int = 5000, blocking: Optional[bool] = None) -> bool:
+    def celebrate(self, move_time: int = 4000, blocking: Optional[bool] = None) -> bool:
```

Comment:
@alexanderEnoch I previously changed move time from 5000 to 4000 to make it the same as wiggle but I didn't know celebrate was implemented for V1. I changed it back so that there is no effect on V1 and V2's celebrate is a slightly faster wiggle (until we get something better for it). Does that sound ok?

Reply:
Cool. sounds good